.circleci/config.yml

```diff
@@ -71,8 +71,13 @@ jobs:
           name: Install requirements
           command: pip3 install -r requirements.txt
       - run:
-          name: Run tests
-          command: python3 -m unittest discover tests
+          name: Install dependencies
+          command: pip3 install coverage
```

Comment:
Could this not be bundled with the previous command? If not, maybe updating the name to `Install coverage` might be more appropriate.

Reply:
Hey @hatched, I have changed it to be "Install coverage".

I haven't included this dependency in the requirements.txt because it is only necessary for the CI, I have seen the same configuration in the snapcraft.io, but let me know what you think about this.

Reply:
Sorry I wasn't clear, I meant `pip3 install -r requirements.txt && pip3 install coverage` 

But this is fine, thanks for the name update.